### ISSUE TITLE
hover does not show when jumping between action bar sections

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#199308]
Testing #199199

1. F1 > focus activity bar
2. Move focus using up/down arrows
3. Hover gets nicely rendered. Hover is not rendered when the focus is moved for the first time to the Account item. Or the first time the focus goes from bottom items to top items 🐛 

These are two actions bars, the top and the bottom one. And it seems like on focus move between these two action bars we are not rendering the hover.

**Comment by meganrogge** [comment]
what am I missing here?

https://github.com/microsoft/vscode/assets/29464607/3799ffa9-57cb-4639-b835-52d3a9ccdf48

**Comment by meganrogge** [comment]
Oh, @isidorn , I think this is not a bug, but a feature request. 

The hover works when you focus the next toolbar item via either `Shift` or `Shift+Tab`. 

Arrow keys will not (and have never) allowed for navigation between those two toolbars. Thus, the focus does not move and the hover closes, as designed, when you try to use an arrow key to jump that boundary.

**Comment by meganrogge** [comment]
I can reproduce when I disable screen reader mode. When in screen reader mode, they have to be tabbed between.

**Comment by isidorn** [comment]
@meganrogge I think it is fine to not fix this as a candidate. But instead do it as part of the December milestone.
You can also discuss in Redmond standup. Thanks 🙏

**Comment by ulugbekna** [comment]
@isidorn as far as I understand the fix is already in. It would be great if you could verify the fix :-)

**Comment by meganrogge** [comment]
Since this is in the release notes, we decided it should be fixed.

**Comment by isidorn** [comment]
I can not repro an original issue, but the behaviour has now degraded more. Here are my steps:

macOS. Latest VS Code insiders. Turn on screen reader mode

1. Focus into activity bar
2. Press `down`. You can not navigate to the bottom activity bar. Focus gets stack on the last item in the top group.

Reopening. Let me know if I am doing something wrong. Thank you

**Comment by meganrogge** [comment]
https://github.com/microsoft/vscode/issues/200165
